server/templates/transcribe.html

```diff
@@ -238,6 +244,21 @@
             submitCore(false /* skipped */ );
         }
 
+        darkModeButton.onclick = function () {
+            const current_background_color = container.style.backgroundColor;
```

Comment:
Can you add "let isDarkMode" in the lets section, and use it instead?
Code would be nicer.

Reply:
no need, i used the switch's built-in `checked` field.